metrics/internal/cache/rbd-mirror.go

```diff
@@ -165,42 +163,32 @@ func initCeph(kubeclient clientset.Interface, cephClusterNamespace, cephAuthName
 		return cephMonitorConfig{}, fmt.Errorf("failed to get ceph user key in namespace %q", namespace)
 	}
 
-	configmap, err := kubeclient.CoreV1().ConfigMaps(cephAuthNamespace).Get(context.TODO(), "rook-ceph-csi-config", metav1.GetOptions{})
+	configmap, err := kubeclient.CoreV1().ConfigMaps(cephAuthNamespace).Get(context.TODO(), "ceph-csi-config", metav1.GetOptions{})
 	if err != nil {
 		return cephMonitorConfig{}, fmt.Errorf("failed to get configmap in namespace %q: %v", namespace, err)
 	}
 
-	data, ok := configmap.Data["csi-cluster-config-json"]
+	data, ok := configmap.Data["config.json"]
```

Comment:
For the new config map, `ceph-csi-config`, just make sure that this data-key, `config.json`, is correct?